pkg/util/winutil/pdhutil/pdhcounter.go

```diff
@@ -133,20 +145,40 @@ func (counter *pdhCounter) countInitError(err error) error {
 	return counter.initError
 }
 
+func (counter *pdhCounter) Remove() error {
+	if counter.handle == PDH_HCOUNTER(0) {
+		return fmt.Errorf("Counter is not initialized")
```

Comment:
Nit: I suspect we aren't linting these files - would've called out lowercasing errors?

Reply:
they should be getting linted, but I'm really not familiar with that process. The CI passed, here's a manual run. does that look right?

```
C:\dev\go\src\github.com\DataDog\datadog-agent>inv -e test --targets=.\pkg\util\winutil\pdhutil
--- Flavor base: golangci_lint
----- Module 'C:\dev\go\src\github.com\DataDog\datadog-agent'
--- Setting rtloader paths to lib:C:\dev\go/src/github.com/DataDog/datadog-agent\dev\lib | header:C:\dev\go/src/github.com/DataDog/datadog-agent\dev\include | common headers:
running golangci on .\pkg\util\winutil\pdhutil
cd C:\dev\go\src\github.com\DataDog\datadog-agent && golangci-lint run --timeout 15m0s --build-tags "cri process containerd orchestrator python kubelet ec2 clusterchecks etcd jmx docker apm consul secrets gce zlib kubeapiserver otlp fargateprocess zk" .\pkg\util\winutil\pdhutil/..
golangci-lint found no issues
```